test/UtilsTest.cpp

```diff
@@ -117,3 +117,46 @@ TEST(UtilsTest, ServerTime) {
     }
     tzset();
 }
+
+TEST(UtilsTest, FormatTime) {
+    struct timeTest {
+        int sec, usec;
+        CString sResultL, sResultN, sResult3N;
+    };
+    timeTest aTimeTests[] = {
+        {42,  12345, "42.012", "42.012345000", "42.012"}, // leading zeroes
+        {42, 999999, "42.999", "42.999999000", "42.999"}, // (no) rounding
+        {42,      0, "42.000", "42.000000000", "42.000"}, // no tv_usec part
+    };
```

Comment:
https://github.com/google/googletest/blob/master/googletest/docs/AdvancedGuide.md#value-parameterized-tests

Reply:
This test case is based on the one in `UserTest.cpp`.

I do not see any existing instances of parametrized tests in ZNC's codebase.

Reply:
That's not an excuse :P
UserTest.cpp can be fixed as well... though not in this pull request, of course.